ontology/yaml/resources/LIGHTING/entity_types/LGRP.yaml

```diff
@@ -225,6 +225,19 @@ LGRP_OS_IM_LSC_LC_PWM_SCZ:
   - PWM
   - SCZ
 
+LGRP_OS_LSC_ENM:
+  description: "Standard Lighting fixture group with occupancy based,area switch status and energy monitoring."
+  is_canonical: true
+  implements:
+  - LGRP
+  - OS
+  - LSC
+  - ENM
+  opt_uses:
+  - total_lamp_failed_counter
+  - wireless_device_failed_counter
+  - communication_failed_counter
```

Comment:
Is this meaning that the lighting group is not communicating, or that the devices within the lighting group arent communicating? If the latter, please use `total` as well.

Reply:
Yes it is the latter one. Wireless device failed counter is the total number of  wireless devices failed in the lighting group and communication failed counter is the total number of devices are not able to communicate in the lighting group.
PS: Added the total for both the fields.